[DOCS] Known issue 2.44.1 (PMM-13956) (#3968)
percona/pmm: documentation/docs/release-notes/2.44.1.md

Fixed [CVE-2023-45288](https://security.snyk.io/vuln/SNYK-GOLANG-GOLANGORGXNETHTTPHTTPPROXY-9058601), a Server-Side Request Forgery (SSRF) vulnerability, by updating the `golang.org/x/net/http/httpproxy` package to version 0.36.0 across multiple components in the PMM Client Docker image. 

This prevents potential attacks that could bypass proxy restrictions and strengthens overall security. 

## Known issue: Incorrect status shown for ProxySQL exporter

> The ProxySQL exporter incorrectly shows a **Done** status instead of **Running** after being added for monitoring.

Reviewer: ```suggestion
The ProxySQL exporter transitions to the **Done** status instead of **Running** after being added for monitoring.
```

Reviewer: The status it shows is correct and due to a failure in processing an empty value of `--log.level`.
